cypress/e2e/data-import/log-details/search-panel-collapsed.cy.js

```diff
@@ -0,0 +1,65 @@
+import { DevTeams, Permissions, TestTypes } from '../../../support/dictionary';
+import TopMenu from '../../../support/fragments/topMenu';
+import Users from '../../../support/fragments/users/users';
+import DataImport from '../../../support/fragments/data_import/dataImport';
+import Logs from '../../../support/fragments/data_import/logs/logs';
+import LogsViewAll from '../../../support/fragments/data_import/logs/logsViewAll';
+import TopMenuNavigation from '../../../support/fragments/topMenuNavigation';
+import DataExportLogs from '../../../support/fragments/data-export/dataExportLogs';
+
+describe('data-import', () => {
+  describe('Log details', () => {
+    let user;
+
+    before('Create test data', () => {
+      cy.createTempUser([
+        Permissions.moduleDataImportEnabled.gui,
+        Permissions.dataExportEnableApp.gui,
+      ]).then((userProperties) => {
+        user = userProperties;
+      });
+    });
+
+    after('Delete test data', () => {
+      cy.getAdminToken().then(() => {
+        LogsViewAll.expandButtonClick();
+        Users.deleteViaApi(user.userId);
+      });
+    });
+
+    it(
+      'C357057 Check that "Search/Filter" panel on the  "View all" page is saved collapsed state after returning from another application. (folijet) (TaaS)',
+      { tags: [TestTypes.extendedPath, DevTeams.folijet] },
+      () => {
+        // #1 Go to the "Data import" app
+        // The "Data import" page is displayed
+        cy.login(user.username, user.password, {
```

Comment:
I'd recommend to move login action to precondition

Reply:
fixed